civis/tests/test_parallel.py

```diff
@@ -376,8 +376,9 @@ def test_result_success(mock_civis):
     # Test that we can get a result back from a succeeded job.
     callback = mock.MagicMock()
     mock_civis.io.civis_to_file.side_effect = make_to_file_mock('spam')
-    fut = ContainerFuture(1, 2, client=mock.MagicMock())
-    fut.set_result(Response({'state': 'success'}))
+    mock_client = create_client_container_mock(1, 2, state='success',
```

Comment:
`ContainerFuture` now has a callback during init, so we can't get away with initializing and setting a result directly during testing. I got around this by moving the container-focused client mock from CivisML to the general tests and using it when initializing the future.